docs/topic-guides/native-api.txt

```diff
@@ -0,0 +1,88 @@
+.. _native-api:
+
+Native API (experimental)
+=========================
+
+.. warning:
+
+    The Native API is currently an experimental API.
+
+The Native API allows developers to seamlessly move between training in local
+Python scripts and training at cluster-scale on a Determined cluster. It also
+provides an interface to train ``tf.keras`` and ``tf.estimator`` models using
+idiomatic framework patterns, reducing (or eliminating) the effort to port
+model code for use with Determined.
+
+In this guide, we'll cover what happens under the hood when a user submits an
+experiment to Determined using the Native API. This topic guide is meant as
+deep-dive for advanced users -- for a tutorial on getting started with the
+Native API, see :ref:`tutorials_native-api`.
+
+Life of a Native API Experiment
+===============================
```

Comment:
got it. But they are rendered at the same level as other titles. 

The list is as below:

```
Optimizing Distributed and Parallel Training
Reproducibility
Scheduling
Terminology and Concepts
User Interfaces
YAML Topic Guide
Native API (experimental)
Life of a Native API Experiment
High-Level Training Loop Support
Integration with the Trial API
```

Seems weird if you click in "Integration with the Trial API" since it's at the first level and find it's actually about Native API. "Integration with the Trial API" alone seems to lose the context of using Native API.

Reply:
My bad, you're totally right. I meant to make them `----`, indexed on the topic guide itself but not in the topic guide `index`. modified.